lib/wallaby/query.ex

```diff
@@ -98,24 +98,26 @@ defmodule Wallaby.Query do
           | :option
           | :select
           | :file_field
+          | :attribute
   @type attribute_key_value_pair :: {String.t(), String.t()}
   @type selector ::
           String.t()
-          | :attribute_key_value_pair
+          | attribute_key_value_pair()
   @type html_validation ::
           :bad_label
           | :button_type
           | nil
   @type conditions :: [
           count: non_neg_integer,
-          text: String.t(),
+          minimum: non_neg_integer,
+          maximum: non_neg_integer,
+          text: String.t() | nil,
           visible: boolean() | :any,
           selected: boolean() | :any,
-          minimum: non_neg_integer,
-          at: pos_integer
+          at: integer | :all
```

Comment:
Should this also be a `non_neg_integer`?

```suggestion
          at: non_neg_integer | :all
```

Here's the relevant test coverage. It seems like -1 isn't valid...

https://github.com/elixir-wallaby/wallaby/blob/bf59c62493469c40bce7467cab0cc6f5aa0a1289/integration_test/cases/query_test.exs#L122-L145

Reply:
I would think that `-1` returns the last element, but maybe we don't support it right now. Looks like the test catches that, you're probably right. I'll revert.